src/dotnet/commands/dotnet-build/LocalizableStrings.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Microsoft.DotNet.Tools.Build
+{
+    internal class LocalizableStrings
```

Comment:
Nit: should be internal static class, but it doesn't matter since these files won't live long.